arm/avhclient/aws_backend.py

```diff
@@ -745,7 +745,8 @@ def run_commands(self, cmds: List[str]):
             # commands which do not need to go to INFO
             commands = [
                 f"runuser -l ubuntu -c 'aws s3 cp s3://{self.s3_bucket_name}/{shfile.name} "
-                f"{self.AMI_WORKDIR}/{shfile.name} && chmod +x {self.AMI_WORKDIR}/{shfile.name}'"
+                f"{self.AMI_WORKDIR}/{shfile.name} --region {os.environ['AWS_DEFAULT_REGION']} && "
```

Comment:
We should move `os.environ['AWS_DEFAULT_REGION']` into a property so that it can be overwritten from command line similar to `self.AMI_WORKDIR` for instance.